http-server/src/main/java/com/facebook/airlift/http/server/HttpServerConfig.java

```diff
@@ -128,6 +128,7 @@
     private boolean authorizationEnabled;
     private AuthorizationPolicy defaultAuthorizationPolicy = AuthorizationPolicy.ALLOW;
     private Set<String> defaultAllowedRoles = ImmutableSet.of();
+    private boolean ignoringUnsecuredRequest;
```

Comment:
nit: say `allowUnsecureRequestsInAuthorizer` which clarifies that it is the authorizer that we are talking about.